### PR TITLE
TabView accessibility and tab size fixes

### PR DESCRIPTION
**Pull request by teaP**
Fixes the following:
#1508: Added name to close button, remove < and > buttons from the accessibility tree as is standard,
#1481: Reverted change that applied min/max sizes to tab in SizeToContent mode.

### CODE REVIEW

File: dev/TabView/TabViewItem.cpp [hunk 1]
```diff
@@ -30,6 +30,13 @@ void TabViewItem::OnApplyTemplate()
         auto closeButton = GetTemplateChildT<winrt::Button>(L"CloseButton", controlProtected);
         if (closeButton)
         {
+            // Do localization for the close button automation name
+            if (winrt::AutomationProperties::GetName(closeButton).empty())
+            {
+                auto closeButtonName = ResourceAccessor::GetLocalizedStringResource(SR_TabViewCloseButtonName);
```
Comment (StephenLPeters) [inline dev/TabView/TabViewItem.cpp]:
```suggestion
auto const closeButtonName = ResourceAccessor::GetLocalizedStringResource(SR_TabViewCloseButtonName);
```

File: dev/TabView/TabView.cpp [hunk 2]
```diff
@@ -572,6 +569,9 @@ void TabView::UpdateTabWidths()
                 }
                 else if (TabWidthMode() == winrt::TabViewWidthMode::Equal)
                 {
+                    double minTabWidth = unbox_value<double>(SharedHelpers::FindResource(c_tabViewItemMinWidthName, winrt::Application::Current().Resources(), box_value(c_tabMinimumWidth)));
+                    double maxTabWidth = unbox_value<double>(SharedHelpers::FindResource(c_tabViewItemMaxWidthName, winrt::Application::Current().Resources(), box_value(c_tabMaximumWidth)));
```
Comment (StephenLPeters) [inline dev/TabView/TabView.cpp]:
```suggestion
                    auto const minTabWidth = unbox_value<double>(SharedHelpers::FindResource(c_tabViewItemMinWidthName, winrt::Application::Current().Resources(), box_value(c_tabMinimumWidth)));
                    auto const maxTabWidth = unbox_value<double>(SharedHelpers::FindResource(c_tabViewItemMaxWidthName, winrt::Application::Current().Resources(), box_value(c_tabMaximumWidth)));
```

**Comment by StephenLPeters** [comment]
@jevansaks does looking up resources from code behind via the SharedHelpers::FindResource method work properly?

**Comment by jevansaks** [comment]
> @jevansaks Jevan Saks FTE does looking up resources from code behind via the SharedHelpers::FindResource method work properly?

Define "properly"? We generally recommend against it since such lookups don't respond to theme changes and they don't do the same walk as even a {StaticResource} would, I think some dictionaries end up getting skipped.

**Comment by StephenLPeters** [comment]
> > @jevansaks Jevan Saks FTE does looking up resources from code behind via the SharedHelpers::FindResource method work properly?
> 
> Define "properly"? We generally recommend against it since such lookups don't respond to theme changes and they don't do the same walk as even a {StaticResource} would, I think some dictionaries end up getting skipped.

This is what I was worried about, @teaP is there a way to do this resource management form markup?

**Comment by teaP** [comment]
> This is what I was worried about, @teaP Tammy Paine FTE is there a way to do this resource management form markup? I guess the control was already doing the lookup from code behind before this change.

Unfortunately, I don't think so. I need to do math with those numbers. I've definitely already noticed that since it pulls the resources from Application.Current, if you have a scoped resource dictionary in your tree with those resources, it doesn't pick them up (and that's the case for all such lookups where we do this).

**Pull request merged**